frontend/src/elements/ListPagination.vue

```diff
@@ -64,15 +60,15 @@ const goto = (index) => {
   emit('update', currentPage.value);
 };
 
-const showPageItem = (p) => pageCount.value < 6 || p == 1 || p == 2 || isVisibleInnerPage(p) || p == pageCount.value - 1;
-const showFirstEllipsis = (p) => pageCount.value >= 6 && currentPage.value > 2 && p == 2;
-const showPageItemLink = (p) => pageCount.value < 6 || p == 1 || isVisibleInnerPage(p);
-const showLastEllipsis = (p) => pageCount.value >= 6 && currentPage.value < pageCount.value - 3 && p == pageCount.value - 1;
+const isVisibleInnerPage = (p) => (currentPage.value === 0 && p === 3)
+    || ((currentPage.value === 0 || currentPage.value === 1) && p === 4)
+    || (p > currentPage.value - 1 && p < currentPage.value + 3)
+    || ((currentPage.value === pageCount.value - 1 || currentPage.value === pageCount.value - 2) && p === pageCount.value - 3)
+    || (currentPage.value === pageCount.value - 1 && p === pageCount.value - 2)
+    || p === pageCount.value;
 
-const isVisibleInnerPage = (p) => (currentPage.value == 0 && p == 3)
-		|| ((currentPage.value == 0 || currentPage.value == 1) && p == 4)
-		|| (p > currentPage.value - 1 && p < currentPage.value + 3)
-		|| ((currentPage.value == pageCount.value - 1 || currentPage.value == pageCount.value - 2) && p == pageCount.value - 3)
-		|| (currentPage.value == pageCount.value - 1 && p == pageCount.value - 2)
-    || p == pageCount.value;
+const showPageItem = (p) => pageCount.value < 6 || p === 1 || p === 2 || isVisibleInnerPage(p) || p === pageCount.value - 1;
+const showFirstEllipsis = (p) => pageCount.value >= 6 && currentPage.value > 2 && p === 2;
+const showPageItemLink = (p) => pageCount.value < 6 || p === 1 || isVisibleInnerPage(p);
+const showLastEllipsis = (p) => pageCount.value >= 6 && currentPage.value < pageCount.value - 3 && p === pageCount.value - 1;
```

Comment:
We may need to clean this up in a post-this-pr PR.

Reply:
Agreed. Past me is to blame here 😅 If you have any improvement suggestions, please just apply them!

Reply:
Haha, well it works for now! I'll give it some thought.